pkg/config/config.go

```diff
@@ -908,6 +908,7 @@ func InitConfig(config Config) {
 	config.BindEnv("orchestrator_explorer.max_per_message")
 	config.BindEnv("orchestrator_explorer.orchestrator_dd_url")
 	config.BindEnv("orchestrator_explorer.orchestrator_additional_endpoints")
+	config.BindEnv("orchestrator_explorer.use_legacy_endpoint")
```

Comment:
You probably want to do `BindEnvAndSetDefault`, and set the default to false. In our experience it makes things cleaner if as many things as possible have a default.